### ISSUE TITLE
Integration Examples Trans

### ISSUE THREAD

**Issue by elbakerino** [ui-schema/demo-cra-ts#7]
There should be created some demo integrations for translations:

- [] i18next 
- [] react-intl

Demos should show ways of:

- [] translating labels
- [] translating values
- [] adding icons
- [] translating errors

Should be done before implementing aria-labeling for lists etc.

Maybe this also helps to get the complexity for building a default `@ui-schema/trans` lib which holds default error messages, compatible to everything.
